pages/map.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  TextField,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  InputAdornment,
+  Card,
+  CardContent,
+  Typography,
+  CardActionArea,
+} from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import Layout from 'components/Layout';
+import { PrismaClient } from '@prisma/client';
+import styles from '../styles/Map.module.css';
+
+const prisma = new PrismaClient();
+
+const Map = dynamic(() => import('../components/Map'), {
+  ssr: false,
+});
+
+type HomeProps = {
+  orgs: PublicOrganization[];
+};
+
+const Home: React.FC<HomeProps> = ({ orgs }) => {
+  const router = useRouter();
+
+  // This is to verify whether or not the current user has a proper session configured to see the page.
+  // Will be implemented in the next PR.
+  // const [session, loading] = useSession();
+
+  return (
+    <Layout>
+      <div className={styles.pageFlex}>
+        <TextField
+          id="outlined-size-small"
+          placeholder="Explore Organizations"
+          fullWidth
+          InputProps={{
+            startAdornment: (
+              <InputAdornment position="start">
+                <SearchIcon />
+              </InputAdornment>
+            ),
+          }}
+          variant="outlined"
+        />
+
+        <div className={styles.pageContent}>
+          <div className={styles.leftCol}>
+            <div className={styles.filters}>
+              <FormControl className={styles.filter} variant="outlined">
+                <InputLabel>Keyword</InputLabel>
+                <Select label="Keyword">
+                  <MenuItem value="">
+                    <em>None</em>
+                  </MenuItem>
+                  <MenuItem value={1}>One</MenuItem>
+                  <MenuItem value={2}>Two</MenuItem>
+                  <MenuItem value={3}>Three</MenuItem>
+                </Select>
+              </FormControl>
+              <FormControl className={styles.filter} variant="outlined">
+                <InputLabel>Keyword</InputLabel>
+                <Select label="Keyword">
+                  <MenuItem value="">
+                    <em>None</em>
+                  </MenuItem>
+                  <MenuItem value={1}>One</MenuItem>
+                  <MenuItem value={2}>Two</MenuItem>
+                  <MenuItem value={3}>Three</MenuItem>
+                </Select>
+              </FormControl>
+              <FormControl className={styles.filter} variant="outlined">
+                <InputLabel>More</InputLabel>
+                <Select label="More">
+                  <MenuItem value="">
+                    <em>None</em>
+                  </MenuItem>
+                  <MenuItem value={1}>One</MenuItem>
+                  <MenuItem value={2}>Two</MenuItem>
+                  <MenuItem value={3}>Three</MenuItem>
+                </Select>
+              </FormControl>
+            </div>
+
+            <div className={styles.cards}>
+              {orgs.length !== 0 ? (
```

Comment:
Is this supposed to be the fix for the typeError that shows up in localhost when there are no valid orgs in the database for mod dash? Because I still get the error if there aren't any valid orgs.

Reply:
this is supposed to be the fix because Prisma should give an empty array instead of undefined

Reply:
i suppose we can add a failsafe by checking if `orgs` actually does exist but if it is undefined, it usually means there's something wrong with your local environment

Reply:
also, i'm assuming these files are temporary or related to another PR? i'm going to delete them for now though